easyrest/models/meta.py

```diff
@@ -49,21 +50,47 @@ def __repr__(self):
         print s
         return ""
 
-    def as_dict(self, exclude=[], include=[]):
-        """Converts model into python dictionary
+    def as_dict(self, exclude=[], include=[], with_relations=[], _isNested=True):
+        """Converts model into python dictionary.
+        Also support relation unpack to one depth
+        To unpack relationship specify its name in with_relations
+        Example: model.as_dict(with_relations=["<relation1>", "<relation2>"])
+        Pls don`t reasing _isNested it uses to limit recursive depth
+        Args:
+            exclude: (list) (str) list of fields to be excluded more priority then include
+            include: (list) (str) list of fields to be included
+            with_relations: (list) (str) list of relationships to include
+            _isNested: (bool) system variable to limit recursion depth
```

Comment:
I absolutely have no idea what `_isNested` does. It needs a better documentation. Also it's name doesn't comply PEP8 rules.